src/Raven.Server/Config/Categories/BackupConfiguration.cs

```diff
@@ -64,6 +63,11 @@ public class BackupConfiguration : ConfigurationCategory
         [ConfigurationEntry("Backup.Azure.Legacy", ConfigurationEntryScope.ServerWideOnly)]
         public bool AzureLegacy { get; set; }
 
+        [Description("Compression algorithm that is used to perform backups (does not apply to snapshot backups) and exports.")]
+        [DefaultValue(BackupCompressionAlgorithm.Zstd)]
+        [ConfigurationEntry("Backup.Compression.Algorithm", ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
1. What would be the name for compression of snapshot backups?
2. Shouldn't we have a dedicated option for export? Do we need to have it configurable for export?

Reply:
1) can do that, yes. That said, tar.zstd may be a better format, no?